solr/core/src/java/org/apache/solr/handler/loader/XMLLoader.java

```diff
@@ -178,13 +183,12 @@ public void load(SolrQueryRequest req, SolrQueryResponse rsp, ContentStream stre
           final byte[] body = IOUtils.toByteArray(is);
           // TODO: The charset may be wrong, as the real charset is later
           // determined by the XML parser, the content-type is only used as a hint!
-          log.trace("body", new String(body, (charset == null) ?
-            ContentStreamBase.DEFAULT_CHARSET : charset));
+          log.trace("body", new String(body, (charset == null) ? ContentStreamBase.DEFAULT_CHARSET : charset));
           IOUtils.closeQuietly(is);
           is = new ByteArrayInputStream(body);
         }
-        parser = (charset == null) ?
-          inputFactory.createXMLStreamReader(is) : inputFactory.createXMLStreamReader(is, charset);
+        parser = (charset == null) ? inputFactory.createXMLStreamReader(is)
```

Comment:
It is very hard to know if this is a real change or a formatting change , looking at the diff. Is it possible to get rid of the irrelevant changes? I'm all for formatting changes as a separate PR